annotations/views/rest_views.py

```diff
@@ -493,8 +501,71 @@ def _relabel(datum):
+                conceptsSet.update(diff)
+            result["identities"] = list(conceptsSet) # replace the identities list
+            if result["identities"]:
+                concepts = result["identities"]
+                uri = result["uri"]
+                if uri in concepts: concepts.remove(uri) # remove original uri from the list if it exists. 
+                new_concepts = []
+                for concept in concepts: # determine if the concept is a viaf or concept power uri
+                    #go through all the concepts and parse xml data for each concept
+                    #then append info to list and then append list to dictionary so that
+                    # This will have to be updated if a new concept Authority is added
+                    hps = re.search( r'www.digitalhps.org', concept, re.M|re.I)
+                    viaf = re.search( r'viaf.org', concept, re.M|re.I)
+                    if hps:
+                        url = concept
+                        data = urllib2.urlopen(url)
+                        tree = ET.parse(data)
+                        root = tree.getroot()
+                        namespace = {'hps': 'http://www.digitalhps.org/'}
+
+                        for entry in root.findall('hps:conceptEntry', namespace):
+                            description1 = entry.find('hps:description', namespace)
+                            name1 = entry.find('hps:lemma', namespace)
+                            concept_description = description1.text
+                            concept_name = name1.text
+
+                        dic = {
+                            'label': concept_name,
+                            'desc': concept_description,
+                            'uri': concept,
+                            'auth': "Concept Power"}
+                        new_concepts.append(dic)
+
+                    elif viaf:
+                        url = concept + '/viaf.xml' # have to append /viaf.xml to viaf url's in order to access xml
+                        data = urllib2.urlopen(url)
+                        tree = e.parse(data)
+                        root = tree.getroot()
+                        namespace = {'foaf': 'http://xmlns.com/foaf/0.1/', 'ns1': 'http://viaf.org/viaf/terms#', 'owl': 'http://www.w3.org/2002/07/owl#', 'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#', 'void': 'http://rdfs.org/ns/void#'}
+                        find = e.XPath("//ns1:text", namespaces = namespace)
+                        concept_name = find(root)[0].text
+                        concept_description = find(root)[0].text
+
+                        dic = {
+                            'label': concept_name,
+                            'desc': concept_description,
+                            'uri': concept,
+                            'auth': "VIAF"}
+                        new_concepts.append(dic)
+                    else:
+                       print "Nothing found!!"
```

Comment:
Nope

Reply:
totally unuseful log output

Reply:
also, if you do want to log something, you should use a proper logger

Reply:
That was just for development and I forgot to remove it.